docs/src/literate/quasi_incompressible_hyperelasticity.jl

```diff
@@ -205,14 +205,14 @@ function assemble_element!(Ke, fe, cell, cellvalues_u, cellvalues_p, mp, ue, pe)
             fe[BlockIndex((ublock), (i))] += ( ∇δui ⊡ ∂Ψ∂F) * dΩ
 
             ∇δui∂S∂F = ∇δui ⊡ ∂²Ψ∂F²
-            @inbounds for j in 1:n_basefuncs_u
```

Comment:
Can remove the `@inbounds` on line 201 as well.

Reply:
Removed this and the one at 222. Thanks! :)